src/oca_github_bot/tasks/migration_issue_bot.py

```diff
@@ -39,7 +39,7 @@ def _set_lines_issue(gh_pr_user_login, gh_pr_number, issue_body, module):
         if added:  # Bypass the checks for faster completion
             lines.append(line)
             continue
-        groups = re.match(rf"^- \[( |x)\] {module}( |\r)", line)
+        groups = re.match(rf"^- \[( |x)\] {module}", line)
```

Comment:
This is not correct at all, as it can match with partial module names. Why is it incorrect? Maybe we can match the end of the line instead.

Reply:
Thanks for the review!
I haven't managed to reproduce exactly the issue #214 because I don't know how the Migration issue is created, but I think this regex might be the cause.

I have found that `\b` is made exactly for matching full words, see https://docs.python.org/3/library/re.html#regular-expression-syntax, can you check if it looks better now?